pravega/pravega
# Issue 4790: Pruning logs for standalone Pravega

Signed-off-by: Abhin Balur <Abhin_Balur@dell.com>

**Change log description**  
- Fixes the extra log output seen if we start Pravega as outlined in [Quick Start Guide](https://pravega.io/docs/nightly/getting-started/quick-start/) or the docker image.

**Purpose of the change**  
  Partial fix towards #4790 

**What the code does**  
   - Uses/corrects the logging configuration done [here](https://github.com/pravega/pravega/pull/5000) for standalone Pravega, which is also used by the docker image.

**How to verify it**  
-  Start Pravega as outlined in [Quick Start Guide](https://pravega.io/docs/nightly/getting-started/quick-start/) or the [docker image](https://pravega.io/docs/nightly/deployment/run-local/#from-docker-image) and we should see a clear set of logs indicating Pravega and its components having started.
